src/control/system/properties_test.go

```diff
@@ -39,3 +43,211 @@ func TestSystem_MgmtProperties(t *testing.T) {
 		t.Fatalf("unexpected attributes (-want +got):\n%s", diff)
 	}
 }
+
+func genTestProps(t *testing.T, min int) (daos.SystemPropertyMap, []*daos.SystemProperty) {
+	t.Helper()
+
+	sysProps := daos.SystemProperties()
+	if len(sysProps) < min {
+		t.Skip("not enough user-visible properties")
+	}
```

Comment:
Would it make sense to fail in this case? Seems odd to me to silently skip a flat-out bad test case.

Reply:
Yeah, I guess we could go either way on that. It would probably be a programming error to somehow wind up with not enough properties from `daos.SystemProperties()`, but I was thinking that maybe we might need to reduce the number because the current set of 2 computed properties is causing a problem or something like that. Eventually I think we'll have plenty of defined system properties so we could change this to a failure.